compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -426,7 +426,7 @@ class MapAllocatorCache {
     //  The difference between the retrieved memory chunk and the request
     //  size is at most MaxAllowedFragmentedPages
     //
-    //  / MaxAllowedFragmentedPages * PageSize \
+    //  / MaxAllowedFragmentedPages * PageSize \.
```

Comment:
Can we change it to `+-` and `-+` like https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/scudo/standalone/primary64.h#L102 ?

Reply:
GTK, done.